views/projectsummary.erb

```diff
@@ -0,0 +1,107 @@
+<head>
+<title>
+  Projects I've Done</title>
+<meta charset = "utf-8">
+</head>
+<header>
+  <h3>Projects</h3>
+</header>
+
+<section>
+<table border="1">
+  <thead>
+    <tr>
+      <th colspan="3">
+        Projects I've Completed in Ada Developers Academy
+      </th>
+    </tr>
+    <tr>
+      <th>
+        Project Name
+      </th>
+      <th>
+        Solo
+      </th>
+      <th>
+        Partner
+      </th>
+    </tr>
+  </thead>
+  <tr>
+    <td style="text-align: center">
+      Calculator
+    </td>
+    <td style="text-align: center">
+      X
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td style="text-align: center">
+      Solar System
+    </td>
+    <td style="text-align: center">
+      X
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td style="text-align: center">
+      Word Guess
+    </td>
+    <td>
+    </td>
+    <td style="text-align: center">
+      X
+    </td>
+  </tr>
+  <tr>
+  <td style="text-align: center">
+    Bank Accounts
+  </td>
+  <td style="text-align: center">
+    X
+  </td>
+  <td>
+  </td>
+</tr>
+<tr>
+  <td style="text-align: center">
+    Scrabble
+  </td>
+  <td>
+  </td>
+  <td style="text-align: center">
+    X
+  </td>
+</tr>
+<tr>
+<td style="text-align: center">
+  FarMar
+</td>
+<td style="text-align: center">
+  X
+</td>
```

Comment:
I had copied and pasted this text from the html project and forgot to reformat it. Will do that in Wave 2.